pom.xml

```diff
@@ -253,6 +253,18 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <!-- TODO Remove this once Hive 2.3.6 released -->
+    <repository>
+      <id>staged</id>
+      <name>staged-releases</name>
+      <url>https://repository.apache.org/content/repositories/staging/</url>
```

Comment:
@srowen, yea almost ready per https://github.com/apache/spark/pull/25443/files#r315437547. Plus, Hive 2.3.6 vote looks going to be open now. After the release, we can switch this snapshot to the official Hive 2.3.6 and I believe we're good to go.

Reply:
FYI, Hive 2.3.6 is being voted at https://www.mail-archive.com/dev@hive.apache.org/msg137043.html